ogr/services/github.py

```diff
@@ -395,6 +395,10 @@ def get_pr_info(self, pr_id: int) -> PullRequest:
         pr = self.github_repo.get_pull(number=pr_id)
         return self._pr_from_github_object(pr)
 
+    def get_all_pr_commits(self, pr_id: int) -> List[str]:
+        pr = self.github_repo.get_pull(number=pr_id)
+        return list([commit.sha for commit in pr.get_commits()])
```

Comment:
```suggestion
        return [commit.sha for commit in pr.get_commits()]
```

Reply:
Good catch. I forgot, that PaginatedList is related only for `pr.get_commit()`.

~~~
In [1]: from ogr import GithubService
In [2]: service = GithubService(token="e620d8c9979385e6b0764b91c9467c6447585519")
In [3]: colin = service.get_project(namespace="user-cont", repo="colin")
In [4]: pr = colin.github_repo.get_pull(number=1)
In [5]: type([commit.sha for commit in pr.get_commits()])
Out[5]: list
In [6]: type(pr.get_commits())
Out[6]: github.PaginatedList.PaginatedList
In [7]:
~~~